generated-platform-project/quarkus-camel/bom/pom.xml

```diff
@@ -398,6 +398,59 @@
         <artifactId>native-protocol</artifactId>
         <version>1.5.0</version>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
+        <artifactId>jackson-datatype-hibernate3</artifactId>
+        <version>2.12.5</version>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
+        <artifactId>jackson-datatype-jaxrs</artifactId>
+        <version>2.12.5</version>
+        <classifier>jakarta</classifier>
```

Comment:
This concerns me. Did you upgrade to Jakarta in Camel?

/cc @aloubyansky

Reply:
Ah it might just be that they are included in the Jackson BOM now. That would explain it, I suppose.

Sorry for the noise :).

Reply:
You are more than welcome. Many thanks for having a look at the PR and sharing explanation :)

Reply:
BTW, just to make sure, we are waiting for the Final before merging this, right?

Reply:
Yes, we can do this. Having a successful CI job to detect more issues would be great though. And then, we have some fixes in camel-quarkus that hopefully could get merged before quarkus 2.8.0.Final.

Reply:
I approved the CI run.

Reply:
@aldettinger @gsmet 
Sorry to interrupt here. Is there any particular reason to not merge this? Meaning except failures on CI build.

@gsmet Is main dedicated to future 2.8.0.Final platform or am I missing something ?

Reply:
On my side, it's possible to merge. The issues could be fixed in a cq 2.8.0-M2 or 2.8.0 (final).

Reply:
Was it test skipping?

Reply:
Yes, it is just FileTest skipping because it is flaky https://github.com/apache/camel-quarkus/issues/3627. I've squashed in a single commit in case we finally merge.

Reply:
@aloubyansky @gsmet  The missing context, is that this is one of our intermediate versions towards camel-quarkus 2.8.0. And we would love to have it merged for nightly builds purpose. We are not used to open this kind of PR.

When Final will be out, we'll do as usually our Final release and open a PR.

If you have any blockers for pushing this PR, we are open to discussion.

/cc @jamesnetherton @ppalaga

Reply:
I have no problem with that @zbendhiba Do you @gsmet?

Reply:
Yes, we will release camel-quarkus 2.8.0 on time for 2.8 Final platform release. 
Thanks, @aldettinger @aloubyansky @gsmet

Reply:
@aloubyansky I was thinking again about this Jackson thing and there's something fishy: we are using Jackson 2.13.1 in Quarkus 2.7 so I have no idea how these weird `jakarta` artifacts end up there in a 2.12 version.

From what I can see, they used to be in the Jackson 2.12.5 BOM (with a specific classifier) and they disappeared later to be replaced by new artifacts.

Not sure I fully grasp things here but I wonder if there's something that needs some tuning in the BOM resolution.

Reply:
The 2.12.5 versions come from `azure-sdk-bom`. The generator is trying to align them but the jakarta classified artifacts don't exist in the later versions, so they are left in their original existing version.